test/unit/ui/map.test.js

```diff
@@ -842,6 +842,44 @@ test('Map', (t) => {
         t.end();
     });
 
+    t.test('#listImages', (t) => {
+        const map = createMap();
+
+        map.on('load', () => {
+            const width = 64;
+            const bytesPerPixel = 4;
+            const data = new Uint8Array(width * width * bytesPerPixel);
+            const imageName = 'gradient';
+
+            for (let x = 0; x < width; x++) {
```

Comment:
For readability let's simplify this test image to be just a very simple small image (e.g. a 4 channel 1x1px or 2x2px image).